lib/fedex/request/shipment.rb

```diff
@@ -68,22 +70,26 @@ def add_label_specification(xml)
       # Callback used after a failed shipment response.
       def failure_response(api_response, response)
         error_message = if response[:process_shipment_reply]
-          [response[:process_shipment_reply][:notifications]].flatten.first[:message]
-        else
-          "#{api_response["Fault"]["detail"]["fault"]["reason"]}\n--#{api_response["Fault"]["detail"]["fault"]["details"]["ValidationFailureDetail"]["message"].join("\n--")}"
-        end rescue $1
+                          [response[:process_shipment_reply][:notifications]].flatten.first[:message]
+                        else
+                          "#{api_response["Fault"]["detail"]["fault"]["reason"]}\n--#{api_response["Fault"]["detail"]["fault"]["details"]["ValidationFailureDetail"]["message"].join("\n--")}"
+                        end rescue $1
         raise RateError, error_message
       end
 
+
+
```

Comment:
As far as I can see, according to the original formatting pattern, the line space was 1 line between function definitions.
I believe that having common formatting pattern improves readability :)